src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/Bau/BauCacheController.cs

```diff
@@ -119,6 +119,20 @@ public async Task<ActionResult> ClearPublicCacheReleases(ClearPublicCacheRelease
             return NoContent();
         }
 
+        [HttpDelete("public-cache/publications")]
```

Comment:
Ah yeah it would make forming the requests easier and it would make unit testing the different variants clearer.

I will leave that for now.

I've just added a new `DELETE /api/bau/public-cache/publication/{publicationSlug}` to be able to clear a publication.
This is in addition to the `DELETE /api/bau/public-cache/publications` already added by this PR.

I know these go against our new policy of only ever deleting from the cache rather than updating it, but in both cases to update rather than delete they would need to use the database to work out what should be updated in terms of publications, releases and latest releases, and also work out what should be removed.

Reply:
Yup understood!